build.gradle.kts

```diff
@@ -24,13 +25,25 @@ group = "org.jmailen.gradle"
 description = projectDescription
 
 object Versions {
-    const val androidTools = "4.2.2"
-    const val jetbrainsAnnotations = "22.0.0"
+    const val androidTools = "7.0.4"
     const val junit = "4.13.2"
-    const val ktlint = "0.43.0"
+    const val ktlint = "0.43.2"
```

Comment:
Yeah, it's probably good to update. I hadn't yet because afaik nothing in 0.43.1 and 0.43.2 impacted kotlinter.
Still waiting on a fix to wildcard imports :/.